app/router.js

```diff
@@ -14,6 +14,7 @@ Router.map(function() {
     });
   });
   this.route('exit');
+  this.route('complete');
```

Comment:
Does this route make sense on its own? Is there an alternate arrangement that would make more sense to the user in the context of this application?

Eg `http://google.com/complete` vs `https://google.com/survey/complete`

Remember that once the product ships, it is much harder to change existing URLs.